testing/testing_index.adoc

```diff
@@ -26,20 +26,20 @@ USER 27
 CMD ["run-mysqld"]
 ----
 
-If we want to test the basic functionality of the container, we do not have to test the RPM functionality. The benefit of using the distribution packaging is that we know testing has already been done during the RPM develpment process. Instead, we need to focus on testing the added scripts instead and the API of the container. The goal is to determine if it works as described. For example, a MariaDB database container, we do not run the MariaDB unit tests in the container. Instead we focus on whether the database is initialized, configured, and responds to commands properly.
+If we want to test the basic functionality of the container, we do not have to test the RPM functionality. The benefit of using the distribution packaging is that we know testing has already been done during the RPM develpment process. Instead, we need to focus on testing the added scripts instead and the API of the container. The goal is to determine if it works as described. For example, for the MariaDB database container, we do not run the MariaDB unit tests in the container. Instead we focus on whether the database is initialized, configured, and responds to commands properly.
 
 === Conventions for Test Scripts
 
 It is good practice to keep the basic sanity tests for the image together with the image sources. For example test/run might be a script, that tests the image specified by the IMAGE_NAME environment variable, so users may specify an image which should be tested.
 
-=== Examples of test scripts
+=== Examples of Test Scripts
```

Comment:
May be some point to MTF http://meta-test-family.readthedocs.io/en/latest/user_guide/index.html